dds/InfoRepo/DCPSInfo_i.cpp

```diff
@@ -463,7 +463,7 @@ OpenDDS::DCPS::RepoId TAO_DDS_DCPSInfo_i::add_publication(
                           , const_cast<DDS::DataWriterQos &>(qos)
                           , const_cast<OpenDDS::DCPS::TransportLocatorSeq &>(transInfo)
                           , csi
-                          , const_cast<DDS::OctetSeq&>(serializedTypeInfo));
+                          , const_cast<DDS::OctetSeq &>(serializedTypeInfo));
```

Comment:
This should actually be `DDS::OctetSeq&`. The file's style is outdated.

Reply:
In general if there is a file or part of a file that goes against the guidelines, don't feel like you have to conform to it.

Reply:
Okay, I wasnt sure.